app/api/datastore/v1/documents.rb

```diff
@@ -27,6 +27,19 @@ class Documents < Base
             :get, **declared(params).symbolize_keys
           ).call
         end
+
+        desc 'List all documents for an application USN.'
+        route_setting :authorised_consumers, %w[crime-apply crime-review]
+        params do
+          requires :usn, type: Integer, desc: 'Application USN.'
+        end
```

Comment:
Should usn be tied to integer?

Reply:
USN are integers. In reality it doesn't matter right now as we only use it to build the bucket prefix which is a string by interpolation but for explicitness I like the parameters declared in the API endpoints as truthful to their underlying data model as possible.